src/components/config-tabs/numerator-groups/NumeratorGroupsTableItem.js

```diff
@@ -0,0 +1,83 @@
+import {
+    ButtonStrip,
+    Button,
+    IconSubtractCircle16,
+    TableCell,
+    TableRow,
+} from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React, { useMemo, useCallback } from 'react'
+import {
+    UPDATE_NUMERATOR_GROUP,
+    useConfigurationsDispatch,
+} from '../../../utils/index.js'
+import { getNumeratorsInGroup } from '../../../utils/numeratorsMetadataData.js'
+
+export const NumeratorGroupsTableItem = ({ numerators, group }) => {
+    const dispatch = useConfigurationsDispatch()
+
+    const numeratorsInGroup = useMemo(
+        () => getNumeratorsInGroup(numerators, group),
+        [numerators, group]
+    )
+
+    const onRemoveNumerator = useCallback(
+        (group, numerator) => {
+            const newGroup = {
+                ...group,
+                members: group.members.filter(
+                    (member) => member !== numerator.code
+                ),
+            }
+
+            dispatch({
+                type: UPDATE_NUMERATOR_GROUP,
+                payload: { updatedGroup: newGroup, code: group.code },
+            })
+        },
+        [dispatch]
+    )
+
+    // check if there are no numerators in group
+    if (numeratorsInGroup.length === 0) {
+        return (
+            <TableRow>
+                <TableCell>No numerators added, please add them.</TableCell>
+            </TableRow>
+        )
+    }
+
+    return (
+        <>
+            {numeratorsInGroup.map((numerator, key) => (
+                <TableRow key={key}>
+                    <TableCell>{numerator.name}</TableCell>
+                    <TableCell>
+                        <ButtonStrip end>
+                            <Button
+                                name="Primary button"
+                                small
+                                onClick={() =>
+                                    onRemoveNumerator(group, numerator)
+                                }
+                                secondary
+                                basic
+                                button
```

Comment:
`button` isn't a valid property on Button component 😀
A number of the other properties here are also not necessary (e.g. `value` (you're not using the value), `basic`/`secondary` should be mutually exclusive (`basic` is what you want, but it's also the default))

Reply:
wait there's `basic` and `secondary` on the same btn? 🙈
let me fix those